src/open_inwoner/components/templates/components/Form/Error.html

```diff
@@ -1,2 +1,10 @@
-{% load notification_tags %}
-{% notifications messages=messages compact=True %}
+{% load i18n button_tags icon_tags button_tags icon_tags notification_tags %}
+
+<div class="notifications notifications__error">
+   <div class="notification notification__error notification--compact">
+    {% for message in messages %}
+        {% icon icon="warning_amber" icon_position="before" outlined=True %}
+        {% notification icon=message.icon icon_position="before" type=message.level_tag message=message.message closable=False compact=compact %}
```

Comment:
Good to swap the `notifications`(meervoud)-tag for plain HTML, but I note it then still uses the same multi-purpose `notification`-tag and styles for the for validation error messages, so I assume only the wrapping div was an issue and not the inner notification component.